lib/modules/manager/gitlabci-include/__snapshots__/extract.spec.ts.snap

```diff
@@ -33,3 +33,20 @@ Array [
   },
 ]
 `;
+
+exports[`modules/manager/gitlabci-include/extract extractPackageFile() extracts multiple embedded include blocks 1`] = `
+Array [
+  Object {
+    "currentValue": "master",
+    "datasource": "gitlab-tags",
```

Comment:
The refs aren't necessarily tags, as far as I know. They could be any sort of git ref, like branch names or commit ids. Can you explain why it's "gitlab-tags" here?

Reply:
I was following the example set in the previous entries within this file. If theres a more specific data source you would like me to use let me know.

Reply:
Yes, the reference can be hashes or branch names too, but this should happen in a separate PR to prevent changes in existing tests.